parser_library/src/workspaces/library_local.h

```diff
@@ -60,16 +62,17 @@ class library_local final : public library, public diagnosable_impl
     bool is_once_only() const override { return false; }
 
 private:
-    file_manager& file_manager_;
+    file_manager& m_file_manager;
 
-    utils::resource::resource_location lib_loc_;
+    utils::resource::resource_location m_lib_loc;
     std::unordered_map<std::string, utils::resource::resource_location, utils::hashers::string_hasher, std::equal_to<>>
-        files_;
-    std::vector<std::string> extensions_;
+        m_files;
+    std::vector<std::string> m_extensions;
     // indicates whether load_files function was called (not whether it was successful)
-    bool files_loaded_ = false;
-    bool optional_ = false;
-    bool extensions_from_deprecated_source = false;
+    bool m_files_loaded = false;
+    bool m_optional = false;
+    bool m_extensions_from_deprecated_source = false;
+    const utils::resource::resource_location& m_proc_grps_loc;
```

Comment:
```suggestion
    utils::resource::resource_location m_proc_grps_loc;
```
I assume the idea was that the worspace_configuration will always outlive the library_local?

Reply:
I think so, yeah. Changed